lib/generators/draft/account/account_generator.rb

```diff
@@ -62,32 +62,32 @@ def authentication_routes
 
       route <<-RUBY.gsub(/^      /, "")
 
-        # Routes for signing up
+        # Routes for the #{singular_table_name.humanize} account:
 
-        match("/#{singular_table_name.underscore}_sign_up", { :controller => "#{plural_table_name.underscore}", :action => "new_registration_form", :via => "get"})
+        # Sign up form
+        get("/#{singular_table_name.underscore}_sign_up", { :controller => "#{plural_table_name.underscore}", :action => "new_registration_form" })
         
-        # Routes for signing in
-        match("/#{singular_table_name.underscore}_sign_in", { :controller => "#{singular_table_name.underscore}_sessions", :action => "new_session_form", :via => "get"})
-        
-        match("/#{singular_table_name.underscore}_verify_credentials", { :controller => "#{singular_table_name.underscore}_sessions", :action => "add_cookie", :via => "post"})
-        
-        # Route for signing out
-        
-        match("/#{singular_table_name.underscore}_sign_out", { :controller => "#{singular_table_name.underscore}_sessions", :action => "remove_cookies", :via => "get"})
-        
-        # Route for creating account into database 
+        # Create record
+        get("/insert_#{singular_table_name.underscore}", { :controller => "#{plural_table_name.underscore}", :action => "create"  })
```

Comment:
I like the re-ordering of these routes. I think it could help drive home the point of forms requiring two routes to work better at a glance.